pkg/services/secrets/kvstore/plugin.go

```diff
@@ -137,7 +162,7 @@ func (kv *secretsKVStorePlugin) Rename(ctx context.Context, orgId int64, namespa
 	return err
 }
 
-func (kv *secretsKVStorePlugin) GetAll(ctx context.Context) ([]Item, error) {
+func (kv *SecretsKVStorePlugin) GetAll(ctx context.Context) ([]Item, error) {
```

Comment:
Might be overcomplicating things, but do you think we should use the fallback here? Possibly call both and merge the lists? May not be needed.

Reply:
The GetAll is only called on plugin migration on fallback directly, and on the delete all API endpoint, unless they call the API during the short windown on which WithFallbackEnabled is executing the fallback on GetAll would never get called.

Reply:
gotcha, seems fine as is then